README.md

```diff
@@ -97,5 +110,10 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
 ```
 
--- YOUR ANSWER HERE --
+Bob
+Curly
+1 
+Mo
+1
+
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.